inginious/frontend/plugins/problem_bank/__init__.py

```diff
@@ -26,7 +26,7 @@ def problem_bank_course_admin_menu_hook(course):
         if not plugin_manager.get_user_manager().has_admin_rights_on_course(course):
             return None
         else:
-            return "problem_bank", '<i class="fa fa-database" aria-hidden="true"></i> Problem bank'
+            return "problem_bank", _("<i class='fa fa-database' aria-hidden='true'></i> Problem bank")
```

Comment:
```suggestion
            return "problem_bank", "<i class='fa fa-database' aria-hidden='true'></i> " + _("Problem bank")
```
Split the string and mark only the text as translatable.